app/components/preprint-form-authors.js

```diff
@@ -10,26 +10,158 @@ export default CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
+                this.set('addState', 'searchView');
+            }
+        },
+        // Removes contributor then redraws contributor list view - removal of contributor may change
+        // which additional update/remove requests are permitted.
         removeContributor(contrib) {
             this.sendAction('removeContributor', contrib);
+            this.toggleProperty('removalToggle');
+            this.toggleProperty('permissionToggle');
+            this.toggleProperty('bibliographicToggle');
+            this.removedSelfAsAdmin(contrib, contrib.get('permission'));
+            this.get('contributors').removeObject(contrib);
         },
-        permissionChange(contributor, permission) {
+        // Updates contributor then redraws contributor list view - updating contributor
+        // permissions may change which additional update/remove requests are permitted.
+        updatePermissions(contributor, permission) {
             this.set(`permissionChanges.${contributor.id}`, permission.toLowerCase());
+            this.sendAction(
+                'editContributors',
+                this.get('contributors'),
+                this.get('permissionChanges'),
+                {}
+            );
+            this.set('permissionChanges', {});
+            this.toggleProperty('permissionToggle');
+            this.toggleProperty('removalToggle');
+            this.removedSelfAsAdmin(contributor, permission);
+
         },
-        bibliographicChange(contributor, isBibliographic) {
+        // Updates contributor then redraws contributor list view - updating contributor
+        // bibliographic info may change which additional update/remove requests are permitted.
+        updateBibliographic(contributor, isBibliographic) {
             this.set(`bibliographicChanges.${contributor.id}`, isBibliographic);
-        },
-        updateContributors() {
             this.sendAction(
                 'editContributors',
                 this.get('contributors'),
-                this.get('permissionChanges'),
+                {},
                 this.get('bibliographicChanges')
             );
+            this.set('bibliographicChanges', {});
+            this.toggleProperty('bibliographicToggle');
+            this.toggleProperty('removalToggle');
+        },
+        // There are 3 view states on left side of Authors panel.  This switches to add unregistered contrib view.
+        unregisteredView() {
+            this.set('addState', 'unregisteredView');
+        },
+        // There are 3 view states on left side of Authors panel.  This switches to searching contributor results view.
+        searchView() {
+            this.set('addState', 'searchView');
+            this.set('fullName', '');
+            this.set('email', '');
+        },
+        emptyView() {
+            this.set('addState', 'emptyView');
+            this.set('searchQuery', '');
+            Ember.$('.searchQuery')[0].value = '';
```

Comment:
We try to avoid manual jquery manipulation of the page for a variety of reasons, among them that it locks together markup/class names with controller logic.

If you must do so, consider using `this.$()` instead of `Ember.$`. As far as I can tell, the latter selects from the entire page (and any other things outside this component instance that use the same class). The former should select only things in or under the container for this component, which is more specific.